src/session_builder.c

```diff
@@ -24,6 +26,16 @@ struct session_builder
     signal_context *global_context;
 };
 
+signal_protocol_store_context *session_builder_get_store(session_builder *session_builder)
+{
+    return session_builder->store;
+};
+
+const signal_protocol_address *session_builder_get_remote_address(session_builder *session_builder)
```

Comment:
Do we really need this function? I think other tests just get directly `&bob_address`

Reply:
removed this function and referenced `&bob_address` directly